actor/builtin/miner/miner.go

```diff
@@ -630,13 +632,21 @@ func (ma *Actor) VerifyPieceInclusion(ctx exec.VMContext, commP []byte, sectorID
 		// Verify proof proves CommP is in sector's CommD
 		var typedCommP types.CommP
 		copy(typedCommP[:], commP)
-		valid, err := verifyInclusionProof(typedCommP, commitment.CommD, proof)
-		if err != nil {
-			return nil, err
+
+		req := verification.VerifyPieceInclusionProofRequest{
+			CommD:               commitment.CommD,
+			CommP:               typedCommP,
+			PieceInclusionProof: proof,
+			PieceSize:           pieceSize,
+			SectorSize:          state.SectorSize,
 		}
 
-		if !valid {
-			return nil, errors.NewRevertError("invalid inclusion proof")
+		res, err := (&verification.RustVerifier{}).VerifyPieceInclusionProof(req)
```

Comment:
FYI in parallel work @acruikshank has been working to abstract the rust verifier out of the actor code, instead exposing methods on the VMContext.